Yona/Yona/Others/SMSValidPasscodeLogin/Login/LoginViewController.swift

```diff
@@ -9,15 +9,19 @@
 import UIKit
 import LocalAuthentication
 
-class LoginViewController: LoginSignupValidationMasterView, AppLifeCylcleConsumer {
-    var loginAttempts:Int = 1
-    fileprivate var totalAttempts : Int = 3
+class LoginViewController: LoginSignupValidationMasterView {
+    
     @IBOutlet weak var bottomSpaceContraint: NSLayoutConstraint!
     @IBOutlet var pinResetButton: UIButton!
     @IBOutlet var closeButton: UIBarButtonItem?
     @IBOutlet var accountBlockedTitle: UILabel?
+    
+    var loginAttempts:Int = 1
```

Comment:
Does this number survive an app restart? Or can the user do an attempt, close the app and do another, for as many times as they want?

Reply:
Code written such a way that **loginAttempts** number won't survive if user restart the app.

Reply:
Then please create a ticket to correct that in a next release.  Talk to @Sivateja-CMI too, to see whether the ticket should be cloned for Android.

Reply:
Ok sure. Is this need to updated to wiki or it's already there?